src/schemas/projectConfig.schema.ts

```diff
@@ -60,8 +60,8 @@ const projectConfigJsonSchema = {
           title: 'StatusProviderName',
           description: 'Name of the status provider',
           type: 'string',
-          enum: ['zeus', 'github'],
-          tsEnumNames: ['Zeus', 'Github'],
+          enum: ['github'],
+          tsEnumNames: ['Github'],
```

Comment:
If an option only has one option, and it's required, is it an option? 🤔

Reply:
Good point but I think it is good to keep this for the reasons below:
1. There's nothing stopping us from adding more status providers (GitLab, GCB etc.)
2. Removing the option would require another sweeping config change